### ISSUE TITLE
Support for Composite Slices.

### ISSUE THREAD

**Issue by smd686s** [prismicio/ruby-kit#73]
Add support for Composite Slices.  See the javascript library for example here:

https://github.com/prismicio/javascript-kit/blob/master/docs/fragments.js.html#L1027

**Comment by CMessinides** [comment]
In the meantime, here's a quick shim to make the current kit work with composite slices. I haven't tested it with any sort of rigor, but it's been working for me so far. Just include this code after requiring the kit:
```ruby
module Prismic
  module JsonParser
    class << self
      
      def slices_parser(json)
        slices = []
        json['value'].each do |data|
          repeat = { 'value' => data['repeat'][0].empty? ? [] : data['repeat'] }
          non_repeat = { 'value' => data['non-repeat'].empty? ? [] : [data['non-repeat']] }
          slices << Prismic::Fragments::Slice.new(data['slice_type'], data['slice_label'], group_parser(repeat), group_parser(non_repeat))
        end
        Prismic::Fragments::SliceZone.new(slices)
      end
    end
  end

  module Fragments
    class Slice
      attr_accessor :repeat
      attr_accessor :non_repeat
      def initialize(slice_type, slice_label, repeat, non_repeat)
        @slice_type = slice_type
        @slice_label = slice_label
        @repeat = repeat
        @non_repeat = non_repeat
      end

      def as_text
        @non_repeat.as_text + ' ' + @repeat.as_text
      end

      def as_html(link_resolver=nil)
        classes = ['slice']
        unless (@slice_label.nil?)
          classes.push(@slice_label)
        end
        %[<div data-slicetype="#{@slice_type}" class="#{classes.join(' ')}">#{@non_repeat.as_html(link_resolver)}#{@repeat.as_html(link_resolver)}</div>]
      end
    end
  end
end
```
This changes the integration of slices. To access the content of a slice, call `slice.repeat` or `slice.non_repeat` instead of `slice.value`. Both `slice.repeat` and `slice.non_repeat` return an instance of `Prismic::Fragments::Group`.

It's ugly as all get out, but if you need a quick fix, it should work. No warranties and YMMV. 🚗

**Comment by arnaudlewis** [comment]
it's fixed in v1.5.0 @smd686s, @CMessinides
